src/The_Cartridge_Header.md

```diff
@@ -1,65 +1,74 @@
+If it doesn't, the boot ROM **locks itself up**.
 
-In older cartridges this area has been part of the Title (see above), in
-newer cartridges this area contains an 4 character uppercase
-manufacturer code. Purpose and Deeper Meaning unknown.
+CGBs and later devices [only check the top half of the logo](Power_Up_Sequence.html?highlight=half#behavior) (the first $18 bytes).
 
-## 0143 - CGB Flag
+## 0134-0143 — Title
 
-In older cartridges this byte has been part of the Title (see above). In
-CGB cartridges the upper bit is used to enable CGB functions. This is
-required, otherwise the CGB switches itself into Non-CGB-Mode. Typical
-values are:
-```
- 80h - Game supports CGB functions, but also works on old Game Boys.
- C0h - Game works on CGB only (physically the same as $80).
-```
-Values with Bit 7 set, and either Bit 2 or 3 set, will switch the
-Game Boy into a special non-CGB-mode called "PGB mode".
+These bytes contain the title of the game in upper case ASCII.
+If the title is less than 16 characters long, the remaining bytes should be padded with `$00`s.
+
+Parts of this area actually have a different meaning on later cartridges, reducing the actual title size to 15 ($0134–$0142) or 11 characters ($0134–$013E); see below.
+
+## 013F-0142 — Manufacturer Code
+
+In older cartridges these bytes were part of the Title (see above).
+In newer cartridges they contain a 4-character manufacturer code (in uppercase ASCII).
+Its purpose is unknown.
+
+## 0143 — CGB Flag
+
+In older cartridges this byte was part of the Title (see above).
+CGBs and later models interpret this byte to decide whether to enable Color mode ("CGB Mode"), or to fall back to monochrome compatibility mode ("Non-CGB Mode").
+
+Typical values are:
 
-TODO: research and document PGB modes...
+Value | Meaning
+------|------------------------------------------------------------------
+`$80` | The game supports CGB enhancements, but is backwards compatible with monochrome Game Boys
+`$C0` | The game works on CGB only (the hardware ignores bit 6, so this really functions the same as `$80`)
 
-## 0144-0145 - New Licensee Code
+Values with bit 7 set, and either bit 2 or 3 set, will switch the Game Boy into a special non-CGB-mode called "PGB mode".
 
-Specifies a two-character ASCII licensee code, indicating the company or
-publisher of the game. These two bytes are used in newer games only
-(games that have been released after the SGB has been invented). Older
-games are using the header entry at `$014B` instead.
+::: tip Research needed
+
+PGB modes are not well researched or documented yet.
+Help is welcome!
+
+:::
+
+## 0144-0145 — New Licensee Code
+
+This area contains a two-character ASCII "licensee code" indicating the game's publisher.
+It is only meaningful if the [Old Licensee Code](<#014B — Old Licensee Code>) is exactly $33 (which is essentially all games made after the SGB was released); otherwise, the old code must be considered.
```

Comment:
```suggestion
It is only meaningful if the [Old Licensee Code](<#014B — Old Licensee Code>) is exactly $33 (which is essentially the case for all games made after the SGB was released); otherwise, the old code must be considered.
```